src/common/TALENTS/shaman.ts

```diff
@@ -678,7 +678,7 @@ const talents = {
     manaCost: 10800,
   },
   HEALING_STREAM_TOTEM_RESTORATION_TALENT: {
-    id: 392916,
+    id: 5394,
```

Comment:
this is an auto-generated file and this modification won't be preserved. you need to add an entry to `common/SPELLS/shaman.ts` that looks like:

```ts
HEALING_STREAM_TOTEM: {
  id: 5394,
  name: 'Healing Stream Totem',
  icon: 'inv_spear_04',
},
```